### PR TITLE
Create edenmath latest

### PR DESCRIPTION
**Pull request by ghost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/edenmath.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'edenmath' do
+  version :latest
+  sha256 :no_check
```
Comment (miccal) [inline Casks/edenmath.rb:3]: Please change these to:
```
version '1.2.2'
sha256 'd171214b7b8d8e07cc855cb78b094064bd440d826243170afac3c11b89a6cf7a'
```

File: Casks/edenmath.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  url 'http://www.edenwaith.com/downloads/edenmath.dmg'
```
Comment (miccal) [inline Casks/edenmath.rb:5]: Please add the following `appcast`:
```
appcast 'http://www.edenwaith.com/products/edenmath/history.php'
```

**Comment by ran-dall** [comment]
@miccal Thank you for the review. I was meaning on getting to it later. 🤝

@gottabeleGIT Also you might need a `zap` stanza.

**Comment by miccal** [comment]
No problem @ran-dall, happy to help.

**Comment by gottabeleGIT** [comment]
@ran-dall The app seems to be self-contained as AppCleaner shows no files besides the .app package.

File: Casks/edenmath.rb (hunk 1 above)
Comment (gottabeleGIT) [reply Casks/edenmath.rb]: Done

File: Casks/edenmath.rb (hunk 2 above)
Comment (gottabeleGIT) [reply Casks/edenmath.rb:5]: Done

**Comment by gottabeleGIT** [comment]
Is there anything else that needs to be done to submit this cask?

**Comment by ran-dall** [comment]
> Is there anything else that needs to be done to submit this cask?

@gottabeleGIT I have to run it locally.

**Comment by ran-dall** [comment]
@gottabeleGIT Sorry for the delay, had some busy work get in the way. This all looks good. Merging...

**Pull request merged**